src/main/java/com/bakdata/common_kafka_streams/KafkaStreamsApplication.java

```diff
@@ -85,12 +85,15 @@
     @CommandLine.Option(names = "--streams-config", split = ",", description = "Additional Kafka Streams properties")
     private Map<String, String> streamsConfig = new HashMap<>();
 
-    @CommandLine.Option(names = "--input-topic", description = "Input topic")
-    protected String inputTopic = "";
+    @CommandLine.Option(names = "--input-topics", description = "Input topics")
```

Comment:
Please add a separator for this list, e.g., `,`. And add a test to verify that two input topics are correctly parsed.